cmd/plugin.go

```diff
@@ -215,32 +223,32 @@ func runPluginInstallCmd(cmd *cobra.Command, args []string) {
 
 	// a leading blank line - since we always output multiple lines
 	fmt.Println()
-
 	progressBars := uiprogress.New()
 	installWaitGroup := &sync.WaitGroup{}
 	dataChannel := make(chan *display.PluginInstallReport, len(plugins))
 
-	progressBars.Start()
-
+	if !silent {
+		progressBars.Start()
+	}
 	for _, pluginName := range plugins {
 		installWaitGroup.Add(1)
 		bar := createProgressBar(pluginName, progressBars)
-		go doPluginInstall(ctx, bar, pluginName, installWaitGroup, dataChannel)
+		go doPluginInstall(ctx, bar, pluginName, installWaitGroup, dataChannel, silent)
```

Comment:
Do we need to send down the `silent` flag here? Can we just ignore the publications in the `dataChannel` if the flag is set? That will consolidate the `silent` logic in one place.